NMS-13730: add discovery config docs from wiki (#3838)
OpenNMS/opennms: docs/modules/operation/pages/provisioning/directed-discovery.adoc

. Repeat for each node you want to add.
. Click *Return* to view the list of nodes you have added.
. Click *Synchronize* to provision them to the {page-component-title} database.

[[requisition-new-suspect]]
== Add nodes via newSuspect events

> The `$\{OPENNMS_HOME}/bin/send-event.pl` script can be used to generate newSuspect events to trigger the Provisiond service to check a given IP address.

Reviewer: ```suggestion
You can use the `$\{OPENNMS_HOME}/bin/send-event.pl` script to generate newSuspect events to trigger the Provisiond service to check a given IP address.
```

Replace `ip-address` in the command below with the address you want to discover.

[source, console]
----
# ${OPENNMS_HOME}/bin/send-event.pl --interface ip-address uei.opennms.org/internal/discovery/newSuspect
----

> NOTE: This will directly add the discovered node into the database.

Reviewer: ```suggestion
NOTE: This adds the discovered node directly into the database.
```

> It is strongly recommended to use <<requisition-add-nodes, requisitions>> when adding individual IP addresses, as it will be easier to manage the nodes over time.

Reviewer: ```suggestion
We strongly recommend that you use <<requisition-add-nodes, requisitions>> when adding individual IP addresses, as it will be easier to manage the nodes over time.
```
